src/lib/Limitation/Templating/LimitationBlockRenderer.php

```diff
@@ -160,4 +148,15 @@ protected function getValueFallbackBlockParameters(Limitation $limitation, array
 
         return $parameters;
     }
+
+    private function getLimitationValueResources(): array
+    {
+        $resources = $this->configResolver->getParameter('limitation_value_templates');
+
+        usort($resources, function ($a, $b) {
```

Comment:
```suggestion
        usort($resources, static function (int $a, int $b): int {
```

Please double check if  the correct type is `int` or `?int`